src/System.Runtime/tests/System/TupleTests.cs

```diff
@@ -180,7 +180,7 @@ public void ToString(string expected)
                 Assert.Equal(expected, Tuple.ToString());
             }
 
-            public void Equals_GetHashCode(TupleTestDriver<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> other, bool expectEqual, bool expectStructuallyEqual)
+            public void Equals_GetHashCode(TupleTestDriver<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> other, bool expectEqual, bool expectStructuallyEqual, bool netfx)
```

Comment:
Do these methods need to be public?  It's not a big deal for a test class like this, but my immediate reaction when I see this is that it's missing [Fact], whereas on a more detailed look I see it's just a helper method for the actual test entrypoints.